config/jobs/gardener-extension-shoot-rsyslog-relp/gardener-extension-shoot-rsyslog-relp-build-images.yaml

```diff
@@ -0,0 +1,48 @@
+postsubmits:
+  gardener/gardener-extension-shoot-rsyslog-relp:
+  - name: gardener-extension-shoot-rsyslog-relp-build-images
+    cluster: gardener-prow-trusted
+    skip_if_only_changed: '^VERSION$'
+    branches:
+    - ^main$
+    annotations:
+      description: Gardener extension shoot-rsyslog-relp image build on main branch
+      testgrid-dashboards: gardener-extension-shoot-rsyslog-relp
+      testgrid-days-of-results: "60"
+    decorate: true
+    max_concurrency: 1
+    spec:
+      serviceAccountName: image-builder
+      containers:
+      - name: image-builder
+        image: eu.gcr.io/gardener-project/ci-infra/image-builder:v20230810-bdb0169
+        command:
+        - /image-builder
+        args:
+        - --log-level=info
+        - --docker-config-secret=gardener-prow-gcr-docker-config
+        - --registry=eu.gcr.io/gardener-project/gardener/extensions
+        - --cache-registry=eu.gcr.io/gardener-project/ci-infra/kaniko-cache
+        - --target=shoot-rsyslog-relp
+        - --target=shoot-rsyslog-relp-admission
+        - --add-version-tag=true
+        - --add-version-sha-tag=true
+        - --add-fixed-tag=latest
+        # image-builder is the pod which is "scheduled" to a node. The pods created by image-builder have an affinity rule
+        # which schedules them to the same node as their parent image-builder. This needs to be done, that PVCs could be mounted
+        # to multiple build pods in parallel.
+        # For a proper scheduling the combined resource requests of all build pods are assigned to this pod, even though it does not
+        # use them. The resource requests of build pods themselves are "0"
+        resources:
+          requests:
+            cpu: 6
+            memory: 2Gi
+      # Node selector is copied to build pods
+      nodeSelector:
+        dedicated: high-cpu
+      # Tolerations are copied to build pods
+      tolerations:
+      - key: "dedicated"
+        operator: "Equal"
+        value: "high-cpu"
+        effect: "NoSchedule"
```

Comment:
I have seen that there is a [publish step](https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/d59c9bccf67d4ed9ae827338748e8ea57bf8ec69/.ci/pipeline_definitions#L12-L27) in concourse which builds images on HEAD updates too.
The prow and concourse build would overwrite each other. Thus, please remove either this prow job or move the concourse image build to `release`.
In the concourse build jobs there is a amd64 and an arm64 build. If you need both of them you should rather keep the concourse job, because we do not build arm64 in prow yet.